src/app/page.tsx

```diff
@@ -46,13 +46,13 @@ export default function Home() {
           className="group rounded-lg border border-transparent px-5 py-2 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
         >
           <h2 className={`mb-2 text-xl font-semibold`}>
-            Om projektet{' '}
+            Om projektet och hur påminnelser fungerar{' '}
```

Comment:
I think this might be a bit wordy to fit nicely, but if it looks fine, why not! (Haven't checked out the code)

Reply:
Agree!